File: ADA_Tema_6/Prueba_SQLiteRoom/app/src/main/java/com/example/prueba_sqliteroom/MainActivity.java
package com.example.prueba_sqliteroom;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;

import com.example.prueba_sqliteroom.room.AppDB;
import com.example.prueba_sqliteroom.room.entidades.Artista;

import java.util.List;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        //Artista newArtista=new Artista("Joaquin Sabina","ESP");
        //AppDB.getAppDB(getApplicationContext()).daoArtista().crearArtista(newArtista);

        Log.d("Spotify_Artista","-----------------------INICIANDO LECTURA------------------");
        List<Artista> artistas=AppDB
                .getAppDB(getApplicationContext())
                .daoArtista()
                .verArtistas();

        for (Artista artista:artistas) {
            Log.d("Spotify_Artista", artista.getNombre());
        }

    }
}

File: ADA_Tema_6/Prueba_Input_Console/app/src/main/java/com/example/prueba_input_console/MainActivity.kt
package com.example.prueba_input_console

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import java.io.InputStream
import java.util.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var tec: Scanner =Scanner(System.`in`)
        var a=tec.nextLine()

        println(":::" + a)
    }
}